tests/test_settings.py

```diff
@@ -666,3 +667,60 @@ def test_update_brand_logo(client: FlaskClient, admin: User) -> None:
     resp = client.get(logo_url, follow_redirects=True)
     # yes this check is ridiculous. why? because we redirect not-founds instead of actually 404-ing
     assert "That page doesn" in resp.text
+
+
+def test_sanitize_input() -> None:
+    from markupsafe import Markup
```

Comment:
import should be at top of file, not in the function

Reply:
5ad4d35